index.html

```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
             </ul>
         </section>
     </div>
+    <div class="works">
+        <section class="portfolio">
+            <div>
+                <img src="Assets/Tonic1.png" alt="Portfolio image">
+            </div>
+            <div>
+                <h1>Tonic</h1>
+                <div class="summary">
+                    <h2 class="canopy">CANOPY</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">Back End Dev</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">2015</h2>                
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="techlist">
+                    <li class="tech">HTML</li>
+                    <li class="tech">CSS</li>
+                    <li class="tech">JavaScript</li>
+                </ul>
```

Comment:
Your design
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/18544c81-73db-4a4f-8634-84f15210d41b)

Figma
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/eee56f1d-60d0-4c53-9b9d-85ae3b5f4e7f)

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please adjust your design to ensure that it is similar to the guideline as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪